pandas/core/generic.py

```diff
@@ -5366,9 +5366,8 @@ def dtypes(self):
         string              object
         dtype: object
         """
-        from pandas import Series  # noqa: F811
-
-        return Series(self._mgr.get_dtypes(), index=self._info_axis, dtype=np.object_)
+        data = self._mgr.get_dtypes()
+        return self._constructor_sliced(data, index=self._info_axis, dtype=np.object_)
```

Comment:
is this overriden by Series?

Reply:
It is in `frame.py` - https://github.com/pandas-dev/pandas/blob/19e8fcc2719575ff0d6a444d035645d009aa2bfa/pandas/core/frame.py#L420

And for `Series` the `dtypes` property is overridden